pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -323,7 +324,14 @@ class RpmFirstStage(Stage):
     """
 
     def __init__(
-        self, remote, repository, deferred_download, skip_types=None, new_url=None, treeinfo=None
+        self,
+        remote,
+        repository,
+        deferred_download,
+        mirror,
```

Comment:
This needs a description in the doc string.
Should the name be more explicit that it's about mirroring metadata?

Reply:
That would be reasonable, yes.  I'll do that.

Reply:
Actually I just removed it entirely. It isn't needed and doesn't belong in the refactor PR since the logic shouldn't change from the user perspective.  But I'll fix it for the next one.